spec/controllers/access_tokens_spec.rb

```diff
@@ -49,13 +93,13 @@
     subject { delete :destroy }
     
     context "when no authorization header provided" do
-      it_behaves_like "unauthorized_requests"
+      it_behaves_like "forbidden_requests"
```

Comment:
updated name to avoid confusion -- unauthorized and unauthenticated started looking too similar to my eyes ; )